src/core/laser_scan_grid_world.h

```diff
@@ -8,15 +14,27 @@
 #include "maps/grid_cell_strategy.h"
 #include "maps/grid_map.h"
 
+/**
+ * \brief Class deriver of World.
```

Comment:
Consider: "[Maintains|Tracks|?] a robots perception of an environment. The environment is represented by a GridMap; laser scan with transformation is expected as sensor data."